services/metadata_service/api/admin.py

```diff
@@ -139,5 +139,5 @@ async def get_authorization_token(self, request):
 
             return web.Response(status=200, body=json.dumps(credentials))
         except Exception as ex:
-            body = {"err_msg": str(ex), "traceback": get_traceback_str()}
+            body = {"message": str(ex), "traceback": get_traceback_str()}
```

Comment:
is this the expected body for the authorization route? the http_500 helper only has a `detail` field for the error message. Consider keeping it consistent with the helper output if possible?

Reply:
I am trying to make them all consistent. Still working on it. After I pushed this, I realized I missed af ew things.